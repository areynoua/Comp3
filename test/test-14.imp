begin
	a := 12
	function @foo(a, b) do
		print(a)
	end;
	@foo(a, b)
end
