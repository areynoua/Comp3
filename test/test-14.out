Unexpected symbol: function with TOS: <ExprArith-p1-j>
